strr-web/lang/en.json

```diff
@@ -279,12 +281,17 @@
     "propertyForm": {
       "subtitle": "Rental Unit Information",
       "nickname": "Nickname (Optional)",
+      "nicknameHelp": "This is only to help you identify your rental unit, especially if you manage multiple properties (e.g., My Guest Suite)",
```

Comment:
I find this text not compact enough. What about: "Nickname to help you identify this rental unit (e.g., My Guest Suite)"

Reply:
Also: Using "My Guest Suite" as an example might be a bit too informal. Consider a more generic or professional example like "Beachside Apartment" or "Downtown Loft" to align better with the short-term rental context.

Reply:
@andyyanggov

Reply:
My thought process: The word "only" is there to assure the user that this field is for their own private use and not part of the approval process. The second part of the sentence is to provide some context as to why the user would need to "identify this rental unit". The example nickname is intentionally more informal/personal to distinguish it from a public listing name.

That said, I'm open to trying Roland's suggestion since it is certainly more compact.

Reply:
@andyyanggov please let me know if there a new content, so I can update it. Thanks

Reply:
Let's keep it the way it is for now since it's already reviewed and approved by Housing.